### PR TITLE
data dict release note updates galileo

### PR DESCRIPTION
**Pull request by bullenc**
Adding data dictionary release notes for Galileo release

### CODE REVIEW

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md
Comment (wwysoc2) [inline docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md:60]: Remove the "items" line and lower the indentation level for the permissible values.  I think this is present in the modified arrays.

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md
Comment (wwysoc2) [inline docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md:110]: Collapse second_gene_symbol into this one.

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md
Comment (wwysoc2) [inline docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md:209]: Add an asterisk to this to make it a list.

**Review by wwysoc2 — CHANGES_REQUESTED** [review]

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md
Comment (bullenc) [reply docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md]: Removed 'items' lines 👍

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md
Comment (bullenc) [reply docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md]: Lumped second_gene_symbol changes with gene_symbol changes

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md
Comment (bullenc) [reply docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md]: Asterisked 👍

**Review by wwysoc2 — APPROVED** [review]
lgtm! Let's wait until we get to QA to merge (in case we have some urgent doc work that needs deploying).

**Review by wwysoc2 — APPROVED** [review]

**Pull request merged**
